src/libawkward/array/VirtualArray.cpp

```diff
@@ -615,13 +621,8 @@ namespace awkward {
     ArrayGeneratorPtr generator = std::make_shared<SliceGenerator>(
                  form, carry.length(), shallow_copy(), slice);
     ArrayCachePtr cache(nullptr);
-    util::Parameters params(parameters_);
```

Comment:
There is one case where the parameters were copied from the parent VirtualArray. I decided it was a non-uniformity and dropped it. We could instead always copy the parameters if that makes more sense.